keyboards/acheron/doddle60/readme.md

```diff
@@ -0,0 +1,29 @@
+# Acheron Aχξρων 1800CP-SM-S-STM32-MX-TH-WI (codename "Austin") QMK firmware
```

Comment:
Couple mentions of Austin, could do with a pass to update them.

Reply:
Weird part is, local files are correct. Maybe forgot to save? No idea 38e8ec232 solves this.